frontend/src/routes/Applications/AdvancedConfiguration.test.tsx

```diff
@@ -370,97 +377,86 @@ describe('Export from application tables', () => {
+    )
   })
 
   test('export button should produce a file for download for channels', async () => {
     render(<TestAdvancedConfigurationPage defaultToggleOption="channels" />)
-    window.URL.createObjectURL = jest.fn()
-    window.URL.revokeObjectURL = jest.fn()
-    const documentBody = document.body.appendChild
-    const documentCreate = document.createElement('a').dispatchEvent
-
-    const anchorMocked = { href: '', click: jest.fn(), download: 'table-values', style: { display: '' } } as any
-    const createElementSpyOn = jest.spyOn(document, 'createElement').mockReturnValueOnce(anchorMocked)
-    document.body.appendChild = jest.fn()
-    document.createElement('a').dispatchEvent = jest.fn()
+    const { blobConstructorSpy, createElementSpy } = getCSVExportSpies()
 
     //download for channels
     await clickByLabel('export-search-result')
     await clickByText('Export all to CSV')
 
-    expect(createElementSpyOn).toHaveBeenCalledWith('a')
-    expect(anchorMocked.download).toContain('table-values')
-
-    document.body.appendChild = documentBody
-    document.createElement('a').dispatchEvent = documentCreate
+    expect(blobConstructorSpy).toHaveBeenCalledWith(
+      [
+        'Name,Namespace,Type,Subscriptions,Clusters,Created\n' +
+          '"ggithubcom-app-samples-ns/ggithubcom-app-samples","default","Git",-,"None","-"',
+      ],
+      { type: 'text/csv' }
+    )
+    expect(getCSVDownloadLink(createElementSpy)?.value.download).toMatch(
+      /^applicationadvancedconfiguration-channels-[\d]+\.csv$/
+    )
   })
 
   test('export button should produce a file for download for placements', async () => {
     render(<TestAdvancedConfigurationPage defaultToggleOption="placements" />)
-    window.URL.createObjectURL = jest.fn()
-    window.URL.revokeObjectURL = jest.fn()
-    const documentBody = document.body.appendChild
-    const documentCreate = document.createElement('a').dispatchEvent
-
-    const anchorMocked = { href: '', click: jest.fn(), download: 'table-values', style: { display: '' } } as any
-    const createElementSpyOn = jest.spyOn(document, 'createElement').mockReturnValueOnce(anchorMocked)
-    document.body.appendChild = jest.fn()
-    document.createElement('a').dispatchEvent = jest.fn()
+    const { blobConstructorSpy, createElementSpy } = getCSVExportSpies()
 
-    // downloads for placements
+    //download for placements
     await clickByLabel('export-search-result')
     await clickByText('Export all to CSV')
 
-    expect(createElementSpyOn).toHaveBeenCalledWith('a')
-    expect(anchorMocked.download).toContain('table-values')
-
-    document.body.appendChild = documentBody
-    document.createElement('a').dispatchEvent = documentCreate
+    expect(blobConstructorSpy).toHaveBeenCalledWith(
+      [
+        'Name,Namespace,Clusters,Created\n' +
+          '"helloworld-simple-placement-3","helloworld-simple-placement-3","1 Remote, 1 Local","4 months ago"',
```

Comment:
@Randy424 Here is another case where we have a relative timestamp in the CSV export data. We'll need to fix this.

Reply:
@KevinFCormier @Randy424  this is causing issues. I created a temporary fix in https://github.com/stolostron/console/pull/4059 but it will need a better, long term solution.

Reply:
Opened a separate ticket for this issue^ https://issues.redhat.com/browse/ACM-15591